frontend-react/src/components/FileHandlers/FileHandlerProgrammersGuideTip.tsx

```diff
@@ -1,20 +1,32 @@
-import { Icon, SiteAlert } from "@trussworks/react-uswds";
+import { Icon } from "@trussworks/react-uswds";
 
 import { USLink } from "../USLink";
 
 export default function FileHandlerProgrammersGuideTip() {
     return (
-        <SiteAlert variant="info" showIcon={false}>
-            <Icon.Lightbulb />
-            <span className="padding-left-1">
-                Pages 18-29 in the{" "}
-                <USLink href="/resources/programmers-guide">
-                    API Programmer’s Guide
-                </USLink>{" "}
-                have the information you need to validate your file
-                successfully. Pay special attention to which fields are required
-                and common mistakes.
-            </span>
-        </SiteAlert>
+        <section
+            data-testid="siteAlert"
+            className="usa-site-alert usa-site-alert--info usa-site-alert--no-heading usa-site-alert--no-icon border-primary-lighter border-width-1px border-solid radius-md"
+            aria-label="Site alert"
+        >
+            <div className="usa-alert">
```

Comment:
Were three <div> tags necessary? Can we just combine them into one?

Reply:
good catch! using the raw USWDS component over the Trussworks one, and yes, without changing the UI, combining those 3 `<div>` with their associated classes works fine.